server/api/observation/observation.model.ts

```diff
@@ -0,0 +1,58 @@
+import { BaseEntity, Column, Entity, JoinColumn, ManyToOne, PrimaryGeneratedColumn, Unique } from 'typeorm';
+import { ObservationType } from './type/observation-type.model';
+
+/**
+ * Observation are created when users self-report COVID-19 symptoms via https://mystatus.mil/
+ * This data is then replicated from the original storage to this entity.
+ */
+@Entity()
+@Unique(['id', 'documentId'])
+export class Observation extends BaseEntity {
+
+  @PrimaryGeneratedColumn()
+  id: number;
+
+  // ID of the original document
+  @Column({ length: 100 })
+  documentId: string;
+
+  /* DOD unique ID that is associated with personnel and their CAC card.
+  We don't maintain the list of these IDs so there is no integrity constraint (foreign key).
+  */
+  @Column({ length: 10 })
+  edipi: string;
+
+  /* timestamp when the observation was reported
+  TODO: https://app.asana.com/0/1198942245525292/1200506195976678/f
+   */
+  @Column({ length: 50 })
+  timestamp: string;
+
+  /* type of the observation symptoms
+  this is a FK to the observation_type table
+   */
+  @ManyToOne(() => ObservationType,
+    observationType => observationType.type,
+    { primary: false, onDelete: 'RESTRICT', eager: true, orphanedRowAction: 'nullify', cascade: false })
+  @JoinColumn({ name: 'type' })
+  @Column({ length: 50 })
+  type: ObservationType;
```

Comment:
Would this be better as a simple Enum type? I think it takes an actual code deployment to add new report/observation types in other parts of the system.

On the other hand, I wonder if this needs more thought in relation to the `ReportSchema` entity? Should it be FK'd there (Instead of the current or enum solutions?) Is a `ReportSchema` actually an `ObservationType`?

Reply:
Will use the report schema: https://app.asana.com/0/1198942245525292/1200516880894434/f